pkg/asset/internal/templates.go

```diff
@@ -431,6 +445,13 @@ spec:
         - name: ssl-host
           mountPath: /etc/ssl/certs
           readOnly: true
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      tolerations:
```

Comment:
This manifest already deinifes a `tolerations` field, can you merge them? @squat